app/models/ability.rb

```diff
@@ -86,7 +86,7 @@ def custom_permissions(user=nil, session=nil)
         end
 
         can :unpublish, MediaObject do |mediaobject|
-          @user.in?(mediaobject.collection.managers)
+          is_editor_of?(mediaobject.collection)
```

Comment:
Does this change mean manager can no longer unpublish? Should this be a ``&&`` statement?